napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1008,6 +1008,69 @@ def cli(self, commands):
             cli_output[py23_compat.text_type(command)] = output
         return cli_output
 
+    def get_environment(self):
+        """
+        Get environment facts.
+
+        power and fan are currently not implemented
+        cpu is using 1-minute average
+        """
+
+        environment = {}
+        cpu_cmd = 'show system resources'
+        mem_cmd = 'show system resources'
```

Comment:
Why run the command two times?

Reply:
Thanks! I should have done one more pass over. I updated and squashed commits into 3ca6028